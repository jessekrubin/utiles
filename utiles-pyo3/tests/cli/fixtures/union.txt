[4188, 3104, 13]
[4192, 2977, 13]
[4192, 3098, 13]
[4192, 2983, 13]
