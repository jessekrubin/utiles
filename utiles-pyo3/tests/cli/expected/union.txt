{"geometry": {"type": "Polygon", "coordinates": [[[4.21875, 44.056011695785244], [4.21875, 44.02442151965933], [4.2626953125, 44.02442151965933], [4.2626953125, 44.056011695785244], [4.21875, 44.056011695785244]]]}, "type": "Feature", "properties": {}}
{"geometry": {"type": "Polygon", "coordinates": [[[4.21875, 43.86621800655638], [4.21875, 43.83452678223682], [4.2626953125, 43.83452678223682], [4.2626953125, 43.86621800655638], [4.21875, 43.86621800655638]]]}, "type": "Feature", "properties": {}}
{"geometry": {"type": "Polygon", "coordinates": [[[4.21875, 40.11168866559596], [4.21875, 40.07807142745009], [4.2626953125, 40.07807142745009], [4.2626953125, 40.11168866559596], [4.21875, 40.11168866559596]]]}, "type": "Feature", "properties": {}}
{"geometry": {"type": "Polygon", "coordinates": [[[4.042968749999999, 39.90973623453719], [4.042968749999999, 39.87601941962115], [4.086914062499999, 39.87601941962115], [4.086914062499999, 39.90973623453719], [4.042968749999999, 39.90973623453719]]]}, "type": "Feature", "properties": {}}
